### PR TITLE
Added statistics per question for a quiz

### PR DESCRIPTION
**Pull request by prathamesh920**

### CODE REVIEW

File: yaksh/templates/yaksh/statistics_question.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+{% block title %} Statistics {% endblock title %}
+
+{% block css %}
+<link rel="stylesheet" href="{{ URL_ROOT }}/static/yaksh/css/question_quiz.css" type="text/css" />
+{% endblock %}
+{% block subtitle %}
+Statistics for {{quiz_name}}
+{% endblock %}
+{% block manage %}
+<p><b>Total number of participants(including multiple attempts): {{ total }}</b></p>
```
Comment (prabhuramachandran) [inline yaksh/templates/yaksh/statistics_question.html]: The multiple attempts is not clear or very meaningful.  For a given attempt for a given question all I want is the number of people who got it right and the total number of people who took the test.

Comment (prabhuramachandran) [inline yaksh/templates/yaksh/statistics_question.html]: The multiple attempts is not clear or very meaningful.  For a given attempt for a given question all I want is the number of people who got it right and the total number of people who took the test.

Comment (prabhuramachandran) [inline yaksh/templates/yaksh/statistics_question.html]: The multiple attempts is not clear or very meaningful.  For a given attempt for a given question all I want is the number of people who got it right and the total number of people who took the test.

Comment (prathamesh920) [inline yaksh/templates/yaksh/statistics_question.html]: Resolved

File: yaksh/views.py [hunk 2]
```diff
@@ -1166,6 +1166,37 @@ def complete(request, reason=None, attempt_num=None, questionpaper_id=None):
...
+        raise Http404('You are not allowed to view this page')
+    papers = AnswerPaper.objects.filter(question_paper_id=questionpaper_id,
+                                        status='completed')
+    total_attempt = papers.count()
+    if total_attempt == 0:
+        return my_redirect('/exam/manage/')
+    questions_answered = []
+    question_stats = {}
+    for paper in papers:
+        questions_answered += paper.questions_answered.split('|')
```
Comment (prabhuramachandran) [inline yaksh/views.py]: Should be a method on the model.

**Comment by prathamesh920** [comment]
statistics_random

**Comment by prabhuramachandran** [comment]
Merging for now but please add tests in a subsequent PR.

**Pull request merged**